app/assets/javascripts/cookiesalert/cookies_alert.js

```diff
@@ -9,30 +9,33 @@ var cookies = {
       cookies.btn = obj.btn || cookies.btn
       cookies.expires = obj.expires || cookies.expires
     }
- 
-    if (Cookies.get('cookies_alert') === undefined) {
+
+    if (Cookies.get('cookies_consent') === undefined) {
       $(cookies.div).css({ visibility : 'visible' })
       $(cookies.btn).click(function () {
         cookies.accept();
+        Cookies.set('cookies_consent', 'true');
         setNonTechnicalCookies();
       })
       $(cookies.denyBtn).click(function () {
         cookies.remove();
-        $(cookies.div).css({ visibility : 'visible' });
+        Cookies.set('cookies_consent', 'false');
+        $(cookies.div).css({ visibility : 'hidden' });
       })
     } else {
       $(cookies.denyBtn).click(function () {
         cookies.remove();
+        Cookies.set('cookies_consent', 'false');
         $(cookies.div).css({ visibility : 'hidden' });
       })
     }
   },
   accept : function () {
-    Cookies.set('cookies_alert','true', { expires : cookies.expires });
+    Cookies.set('cookies_consent','true', { expires : cookies.expires });
     $(cookies.div).css({ visibility : 'hidden' })
   },
   remove : function () {
```

Comment:
rename function to `only_technical_cookies`

Reply:
Which part should be extracted to only_technical_cookies?